actors/builtin/paych/paych_test.go

```diff
@@ -407,42 +421,56 @@ func TestActor_UpdateChannelStateMergeFailure(t *testing.T) {
 		ucp := &UpdateChannelStateParams{Sv: *sv}
 		rt.SetCaller(st1.From, builtin.AccountActorCodeID)
 		rt.ExpectValidateCallerAddr(st1.From, st1.To)
+		rt.ExpectVerifySignature(*ucp.Sv.Signature, actor.payee, voucherBytes(t, &ucp.Sv), nil)
 		rt.ExpectAbort(exitcode.ErrIllegalArgument, func() {
 			rt.Call(actor.UpdateChannelState, ucp)
 		})
+		rt.Verify()
 	})
 }
 
 func TestActor_UpdateChannelStateExtra(t *testing.T) {
-	rt1, actor1, sv1 := requireCreateChannelWithLanes(t, context.Background(), 1)
```

Comment:
Constructing the runtime outside of the `t.Run()` blocks below is invalid since it leads to the wrong `testing.T` being used. There are other places that do this too, but the tests still happen to pass so I didn't push further.

Reply:
Probably worth leaving a `TODO/FIXME` flagging this

Reply:
Turned out there was only one more case, so I fixed it.